core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/autoscales/AutoscaleV4Endpoint.java

```diff
@@ -52,6 +53,22 @@
     @ApiOperation(value = StackOpDescription.PUT_BY_ID, produces = APPLICATION_JSON, notes = Notes.STACK_NOTES, nickname = "putStackForAutoscale")
     void putStack(@PathParam("crn") String crn, @PathParam("userId") String userId, @Valid UpdateStackV4Request updateRequest);
 
+    // Not overloading the regular scaling API since 1) that is public, and 2) stopstart may move into a separate InstancePoolManagementController at some point
+    @PUT
+    @Path("/stack/startNodes/crn/{crn}")
+    @Produces(APPLICATION_JSON)
+    @ApiOperation(value = StackOpDescription.PUT_START_INSTANCES_BY_ID, produces = APPLICATION_JSON,
+            notes = Notes.STACK_NOTES, nickname = "putStackForAutoscaleStartStop")
+    void putStackStartInstances(@PathParam("crn") String crn, @Valid UpdateStackV4Request updateRequest);
+
+    // TODO CB-14929: Remove this API once done with testing, or publish a quick document somewhere on how the put API can be used
+    @GET
+    @Path("/stack/startNodees/crn/{crn}/{userId}")
+    @Produces(APPLICATION_JSON)
+    @ApiOperation(value = "start_nodes_by_count", produces = APPLICATION_JSON, notes = "blah", nickname = "tmpStartNodes")
+    String tmpStartNodes(@PathParam("crn") String crn, @PathParam("userId") String userId,
+            @QueryParam("hostGroup") String hostGroup, @QueryParam("numNodes") Integer numNodes);
```

Comment:
we cannot merge this until this test APIs are here, since removing an existing API is not an easy task and these are callable APIs by the user, so they can easily mess up something

Reply:
Sorry. This should have been removed, will get rid of the temporary APIs.

Reply:
Fixed in the next push. Resolving conversation.